app/services/onlinetesting/Phase1TestService.scala

```diff
@@ -126,6 +126,13 @@ trait Phase1TestService extends OnlineTestService with ResetPhase1Test {
     registerAndInviteForTestGroup(application, getScheduleNamesForApplication(application))
   }
 
+  override def processNextExpiredTest(expiryTest: TestExpirationEvent)(implicit hc: HeaderCarrier, rh: RequestHeader): Future[Unit] = {
```

Comment:
I remember I gave this a thought but you can't know the repo if you place it in OnlineTestService as is the trait that collect all the common functionality regardless of the phase (and repo)